pom.xml

```diff
@@ -122,16 +126,47 @@
 
     <resources>
       <resource>
-        <directory>src/main/resources</directory>
+        <directory>${project.basedir}/src/main/resources</directory>
         <filtering>true</filtering>
       </resource>
     </resources>
 
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>animal-sniffer-maven-plugin</artifactId>
+          <version>1.14</version>
```

Comment:
I add the version to properties as soon as I can.

--- Original Message ---

From: "Matt Bishop" <notifications@github.com>
Sent: April 27, 2015 10:18 PM
To: "qos-ch/slf4j" <slf4j@noreply.github.com>
Cc: "Jeremy Landis" <jeremylandis@hotmail.com>
Subject: Re: [slf4j] Apply Standard Maven Usage (#107)

>          <filtering>true</filtering>
>        </resource>
>      </resources>
>
> +    <pluginManagement>
> +      <plugins>
> +        <plugin>
> +          <groupId>org.codehaus.mojo</groupId>
> +          <artifactId>animal-sniffer-maven-plugin</artifactId>
> +          <version>1.14</version>

Most of the other plugins have their version # propertized.

---
Reply to this email directly or view it on GitHub:
https://github.com/qos-ch/slf4j/pull/107/files#r29209849